src/registries/menus/menu_items_actions.ts

```diff
@@ -715,6 +715,14 @@ export const INSERT_LINK = (env: SpreadsheetChildEnv) => {
   env.model.dispatch("OPEN_CELL_POPOVER", { col, row, popoverType: "LinkEditor" });
 };
 
+export const INSERT_LINK_NAME = (env: SpreadsheetChildEnv) => {
+  const sheetId = env.model.getters.getActiveSheetId();
+  const { col, row } = env.model.getters.getPosition();
+  const cell = env.model.getters.getCell(sheetId, col, row);
+
+  return cell && cell.isLink() ? _lt("Edit link") : _lt("Insert link");
```

Comment:
we have to update translation .pot files

Reply:
I will provide you with the PR for that shortly.

Reply:
Don't worry, I'll do when we update the lib in odoo, in the same PR :wink:

Reply:
Great, Thanks :)